fiftyone/utils/cvat.py

```diff
@@ -3939,6 +3947,48 @@ def _upload_annotations(
             num_uploaded_tags = len(anno_resp["tags"])
             num_uploaded_tracks = len(anno_resp["tracks"])
 
+        server_id_map = self._map_cvat_server_id(anno_resp, attr_id_map)
+
+        return server_id_map
+
+    def _map_cvat_server_id(self, anno_resp, attr_id_map):
+        # Create mapping from CVAT server ID to FiftyOne ObjectId for each
+        # label
+        server_id_map = {}
+        dup_ids = []
+        types = ["shapes", "tags", "tracks"]
+        attr_id_map_rev = {}
+        for class_id, class_attr_map in attr_id_map.items():
```

Comment:
You only care about the `label_id` attribute in this method, so it would be cleaner to just extract its ID here and then use it below (no need to map back to attribute strings later)

Reply:
I think I resolved this. I'm not 100% sure what you meant by this, though.